.github/workflows/release.yml

```diff
@@ -437,13 +437,53 @@ jobs:
         path: |
           signed
 
+# ================================
+#           Validate
+# ================================
+  validate:
+    name: Validate installers
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    needs: [ osx-sign, win-sign, linux-sign ]
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Build & Install Windows
+        if: contains(matrix.os, 'windows')
+        shell: sh
+        run: |
+          exePaths=$(find ./win-sign/signed/*.exe) &&
+          for path in $exePaths; do
+            start "" "$path" /WAIT //SILENT //NORESTART &&
```

Comment:
We probably also want `//VERYSILENT`.

Reply:
👍

Reply:
To clarify - do we want `//VERYSILENT` in addition to `//SILENT` or instead of `//SILENT`?

Reply:
I think `//VERYSILENT` includes `//SILENT`, but not 100% sure. Docs don't make it too clear:

https://jrsoftware.org/ishelp/topic_setupcmdline.htm#SILENT

Reply:
In Git for Windows, we always use `//VERYSILENT` _in addition to_ `//SILENT`, but it might be a "better safe 'n sorry" thing. I have no idea whether `//VERYSILENT` _implies_ `//SILENT`. I guess it's time to consult the source.

*clicketyclick*

Seems that [`//VERYSILENT` sets the mode to something different than `//SILENT`](https://github.com/jrsoftware/issrc/blob/cb9c3e94593724ea062bb3070bb034abdd009652/Projects/Main.pas#L2781-L2784), but it would appear as if `//VERYSILENT` indeed does imply `//SILENT` (there is only [this](https://github.com/jrsoftware/issrc/blob/cb9c3e94593724ea062bb3070bb034abdd009652/Projects/Main.pas#L2786-L2792) and [this](https://github.com/jrsoftware/issrc/blob/cb9c3e94593724ea062bb3070bb034abdd009652/Projects/Main.pas#L3069-L3070) and [this](https://github.com/jrsoftware/issrc/blob/cb9c3e94593724ea062bb3070bb034abdd009652/Projects/Main.pas#L3265) and [this](https://github.com/jrsoftware/issrc/blob/cb9c3e94593724ea062bb3070bb034abdd009652/Projects/Main.pas#L4267-L4278) part of the code, and it all looks as if the `imVerySilent` mode does more than `imSilent`, not different things).

tl;dr it does seem as if `//VERYSILENT` should be enough. To be safe, though, I would use both (I vaguely remember that we had to do this at some stage, probably due to a hence-fixed bug in InnoSetup).

Reply:
Okie doke, I'll add both!